code/modules/mob/dead/dead.dm

```diff
@@ -32,6 +32,7 @@ INITIALIZE_IMMEDIATE(/mob/dead)
 		onTransitZ(old_turf?.z, new_turf?.z)
 	var/oldloc = loc
 	loc = destination
+	SEND_SIGNAL(src, COMSIG_MOVABLE_LOCATION_CHANGE, oldloc)
```

Comment:
Same comment here.

Reply:
instead of an overridden forceMove() its now an overridden abstract_move()

Reply:
and the signal is gone